Channels/Integration/01a_Shopware6.md

```diff
@@ -0,0 +1,172 @@
+  The *Create integration* window is displayed.
+
+  ![Create integration](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/CreateIntegration.png "[Create integration]")
+
+7. Enter a name for the integration in the *Name* field and enable the *Administrator* toggle.
+
+8. Copy the code provided in the *Access key ID* field to your clipboard.
+
+9. Switch to the *Actindo Core1 Platform* and paste the copied access ID from your clipboard in the *Access key ID* field in the *Credentials* section.  
+  
+10. Repeat the steps **8** and **9** for the *Secret access key* field.
+
+11. Switch to your Shopware 6 account and click the [Save integration] button.  
+  The integration is saved. The *Success* pop-up window is displayed.
+
+  ![Integration  created](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/IntegrationCreated.png "[Integration created]")
+
+12. Switch to the *Actindo Core1 Platform* and click the [SAVE] button.  
+  The connection has been created.   
+  The *CONNECTIONS* tab in the *Settings* menu entry of the *Omni-Channel* module is displayed when the connection has been established. The *Shopware 6* connection is displayed in the list of connections.
+
+  ![Shopware 6 connection](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/Connection.png "[Shopware 6]")
+
+12. If necessary, continue to [Configure the Shopware 6 connection](#configure-the-shopware-6-connection).
+
+[comment]: <> (Was genau passiert nach SAVE in Core1? Einige Minuten Zeit/Sync/Platform initialized? Satz aus Shopify genommen.)
+
+
+
+## Configure the Shopware 6 connection   
+
+After the connection to a Shopware 6 shop has been established, further settings can be configured for the connection.
+
+#### Prerequisites
+
+A Shopware 6 connection has been established, see [Create a Shopware 6 connection](#create-a-shopware-6-connection).
+
+#### Procedure
+
+*Omni-Channel > Settings > Tab CONNECTIONS*
+
+![Shopware 6 connection](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/Connection.png "[Shopware 6 connection]")
+
+1. Click the Shopware 6 connection in the list of connections.   
+  The *Edit connection* view is displayed. By default, the *Credentials* tab is selected.
+
+  ![Edit connection credentials](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/EditConnectionCredentials.png "[Edit connection credentials]")
+
+2. Click the *Settings* tab.   
+  The *Settings* tab is displayed. By default, the *Default language* setting is selected.
+
+  > [Info] In Shopware 6 there is a standard language, which is configured during the installation and cannot be subsequently edited. The standard language cannot be detected via API and must be therefore manually configured in the *Actindo Core1 Platform* after the connection has been established. When uploading products, Shopware 6 requires some fields to be filled for the standard language.
```

Comment:
Vielleicht sollte diese Info in den Einführungssatz mit rein? Lass uns drüber sprechen. 
Auf jeden Fall fehlt hier aber der Step, dass ich die Standardsprache einstelle...